src/battle/partner/bombette_6FAD10.c

```diff
@@ -1,11 +1,254 @@
+    BattleStatus* battleStatus = &gBattleStatus;
+    Actor* partnerActor = battleStatus->partnerActor;
+    s32 var0 = get_variable(script, *args++);
+    s32 var1 = 0;
+
+    switch (partnerActor->staticActorData->level) {
+        case 0:
+            if (var0 < 36) {
+                var1 = 1;
+            }
+            else if (var0 < 61) {
+                var1 = 2;
+            }
+            else if (var0 < 81) {
+                var1 = 3;
+            }
+            else if (var0 < 100) {
+                var1 = 4;
+            }
+            else {
+                var1 = 5;
+            }
+            break;
+        case 1:
+            if (var0 < 36) {
+                var1 = 1;
+            }
+            else if (var0 < 61) {
+                var1 = 3;
+            }
+            else if (var0 < 81) {
+                var1 = 4;
+            }
+            else if (var0 < 100) {
+                var1 = 5;
+            }
+            else {
+                var1 = 6;
+            }
+            break;
+        case 2:
+            if (var0 < 36) {
+                var1 = 1;
+            }
+            else if (var0 < 61) {
+                var1 = 3;
+            }
+            else if (var0 < 81) {
+                var1 = 5;
+            }
+            else if (var0 < 100) {
+                var1 = 6;
+            }
+            else {
+                var1 = 7;
+            }
+            break;
+        default:
+            break;
+    }
+
+
+    if (!(var0 < 100)) {
+        script->varTable[0] = 1;
+    } else {
+        script->varTable[0] = 0;
+    }
+
+    script->varTable[15] = var1;
+
+    return 2;
+}
+
+ApiStatus func_8023849C_6FB1AC(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    BattleStatus* battleStatus = &gBattleStatus;
+    Actor* partnerActor = battleStatus->partnerActor;
+    s32 var0 = get_variable(script, *args++);
+    s32 var1 = 0;
+
+    switch (partnerActor->staticActorData->level) {
+        case 1:
+            if (var0 < 36) {
+                var1 = 1;
+            }
+            else if (var0 < 61) {
+                var1 = 3;
+            }
+            else if (var0 < 81) {
+                var1 = 4;
+            }
+            else if (var0 < 100) {
+                var1 = 5;
+            }
+            else {
+                var1 = 6;
+            }
+            break;
+        case 2:
+            if (var0 < 36) {
+                var1 = 1;
+            }
+            else if (var0 < 61) {
+                var1 = 3;
+            }
+            else if (var0 < 81) {
+                var1 = 5;
+            }
+            else if (var0 < 100) {
+                var1 = 6;
+            }
+            else {
+                var1 = 7;
+            }
+            break;
+        default:
+            break;
+    }
+
+
+    if (!(var0 < 100)) {
+        script->varTable[0] = 1;
+    } else {
+        script->varTable[0] = 0;
```

Comment:
I purposely don't call varTable related assignments `TRUE` or `FALSE` anymore, since there were cases before were they actually looked at the number, or suddenly a `2` appeared in another function that called it. 
tl;dr I rather have `0` and `1` instead of calling them `TRUE` and `FALSE` if I am not 100% sure.

Reply:
i mean here there are very clearly two cases